3 Reports.sql

```diff
@@ -0,0 +1,22 @@
+
+--1)We need a list of how many of each pet we currently have.
+select g.pettype, PetCount = count(*)
+from grooming g
+group by g.pettype
+
+--2)we need to see how many customers have multiple pets.
+select g.OwnersName ,PetCount = count(*)
+from grooming g
+group by g.OwnersName
+having count(*) > 1
+
+--3)We need a list of the number of each kind of animal we have, for advertising and hiring purposes
+--How is this question different than question #1?
```

Comment:
You are right, this question should be changed or removed, it is the same as the first, thanks for catching that